extensions/wikia/FacebookClient/scripts/FacebookClient.XFBML.js

```diff
@@ -6,16 +6,17 @@
 mw.hook('wikipage.content').add(function ($content) {
 	'use strict';
 
-	if ((xfbmlTagsOnPage())) {
-		$.loadFacebookSDK().done(renderXFBMLTags);
+	if ((faceBookTagsOnPage())) {
+		$.loadFacebookSDK().done(renderFaceBookTags);
 	}
 
-	function xfbmlTagsOnPage() {
-		var numOfXFBMLTags = $content.find('[data-type="xfbml-tag"]').length;
-		return numOfXFBMLTags > 0;
+	// Checks for XFBML tags or FaceBook Page Plugin tags
+	function faceBookTagsOnPage() {
```

Comment:
I think all instances of "facebook" here should be with a lowercase "b", since that's how they brand themselves.